src/UserService.Mappers/Db/DbUserEducationMapper.cs

```diff
@@ -4,38 +4,52 @@
 using LT.DigitalOffice.UserService.Models.Dto.Requests.User.Education;
 using Microsoft.AspNetCore.Http;
 using System;
+using System.Collections.Generic;
 
 namespace LT.DigitalOffice.UserService.Mappers.Db
 {
-    public class DbUserEducationMapper : IDbUserEducationMapper
+  public class DbUserEducationMapper : IDbUserEducationMapper
+  {
+    private readonly IHttpContextAccessor _httpContextAccessor;
+
+    public DbUserEducationMapper(IHttpContextAccessor httpContextAccessor)
     {
-        private readonly IHttpContextAccessor _httpContextAccessor;
+      _httpContextAccessor = httpContextAccessor;
+    }
 
-        public DbUserEducationMapper(IHttpContextAccessor httpContextAccessor)
-        {
-            _httpContextAccessor = httpContextAccessor;
-        }
+    public DbUserEducation Map(CreateEducationRequest request, List<Guid> imagesIdsForCreate)
+    {
+      if (request == null)
```

Comment:
add not null check for imagesIdsForCreate

Reply:
it can be null + reworked